news/bugfix-3064.md

```diff
@@ -0,0 +1 @@
+`loggen`: Fixed message loss with TLSv1.3.
```

Comment:
I think we should add a more detailed description related to this issue.

Reply:
@alltilla I've opened a PR in your fork, it contains a longer description for this bugfix.
https://github.com/alltilla/syslog-ng/pull/5/files

Reply:
Thanks, merged :)